src/Components/Articles.jsx

```diff
@@ -0,0 +1,40 @@
+import {useEffect, useState} from 'react'
+
+
+const Articles = () => {
+
+    const [articles, setArticles] = useState([])
```

Comment:
I would be tempted to implement an isLoading pattern here as it could take some time before these articles come back